bridge/README.md

```diff
@@ -3,17 +3,86 @@
 The easiest way to deploy from MLFlow to SageMaker
 
 ![build](https://github.com/dominodatalab/domino-research/actions/workflows/bridge.yml/badge.svg?branch=main)
+[![Docker Repository on Quay](https://quay.io/repository/domino/bridge/status "Docker Repository on Quay")](https://quay.io/repository/domino/bridge)
```

Comment:
the leading [ looks like a typo

Reply:
copied straight from Quay. I think its just placing a link inside of the image tag.